app/models/team_zip_export.rb

```diff
@@ -0,0 +1,273 @@
+require 'zip'
+require 'fileutils'
+require 'csv'
+
+class TeamZipExport < ZipExport
+  has_attached_file :zip_file,
+                    path: '/zip_exports/:attachment/:id_partition/' \
+                          ':hash/:style/:filename'
```

Comment:
That's the whole reason to have it there, otherwise it tries to download '/team_zip_exports/...' file, which does not exist. Just to comply with the `ZipExport` download function and to have the same logic.